src/Morphir/Visual/Components/AbstractTreeVisualizer.elm

```diff
@@ -0,0 +1,272 @@
+module Morphir.Visual.Components.AbstractTreeVisualizer exposing (..)
```

Comment:
Is this just a sample app? If so, can you move it to the `tests` folder or within the `cli` folder?